Extensions/old_stats.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE Old Stats **//
-//* VERSION 0.2.2 **//
+//* VERSION 0.3.0 **//
```

Comment:
Should probably be a `MAJOR` increment, since we're drastically changing the functionality here.